tests/test_service.py

```diff
@@ -131,3 +131,70 @@ def test_get_system_information_without_pkg(self):
         cond = (ReportPortalService.get_system_information('pytest')
                 == expected_result)
         assert cond
+
+    @mock.patch("reportportal_client.service._get_data",
+                mock.Mock(return_value={"id": 123}))
+    def test_start_item(self, rp_service):
+        """Test for validate start_test_item.
+
+        :param: rp_service: fixture of ReportPortal
+        """
+        rp_start = rp_service.start_test_item(name="name",
+                                              start_time=1591032041348,
+                                              item_type='STORY')
+        expected_result = dict(json={'name': 'name',
+                                     'description': None,
+                                     'attributes': None,
+                                     'startTime': 1591032041348,
+                                     'launchUuid': 111,
+                                     'type': 'STORY', 'parameters': None,
+                                     'hasStats': True,
+                                     'codeRef': None},
+                               url='http://endpoint/api/v2/project/item',
+                               verify=True)
+
+        rp_service.session.post.assert_called_with(**expected_result)
+        assert rp_start == 123
+
+    start_item_optional = [
```

Comment:
class attributes need to be defined before methods.

Reply:
If we talk about regular class then yes, but this is unit-tests. I just try too keep data near a location where it is used.